ioctl/test_ioctl.c

```diff
@@ -0,0 +1,394 @@
+/*
+ * Phoenix-RTOS
+ *
+ *    Phoenix-RTOS standard library functions tests
+ *    HEADER:
+ *    - ioctl.h
+ *    TESTED:
+ *    - ioctl()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <string.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/ioctl.h>
+#include <phoenix/ioctl.h>
+#include <sys/msg.h>
+#include <signal.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <pthread.h>
+#include <posix/utils.h>
+#include <unity_fixture.h>
+
+static uint32_t port;
+static int fd, dd = 5;
+static volatile int loop_flag = 1;
```

Comment:
name it eg. `thread_running` for better readability (there would be for the extra comment when setting `thread_running = 0`